src/more-or-less/moreOrLess.scss

```diff
@@ -1,6 +1,7 @@
 $expand-button-background: #EAEAEA !default;
 $expand-button-hover-background: #F1F2F3 !default;
 $expand-button-icon-color: #999 !default;
+$expand-button-icon-hover-color: #999 !default;
```

Comment:
I'm following the convention of adding the state (`hover`, `open`, `has-value`....) always at the end of the variable. please replace with `$expand-button-icon-color-hover` :)

PS: sorry about this, I'll write a small readme after this wonderful design week